openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModelTest.java

```diff
@@ -236,4 +249,115 @@ private boolean compareDoubles(final double a, final double b, final int exclude
         );
     }
 
+    /**
+     * Compares model files in the standard_code_models/ and new_code_models/.
```

Comment:
This is a bit confusing.
Are you comparing those folders with each other?
Or models within each folder, e.g.,  standard_code_models/ has two models, comparing them. new_code_models/ has 3 models, comparing them.

Reply:
You compare files with the same name between the two folders.

I accept suggestions for rewording :)